gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/catalog/GfxdSystemProcedures.java

```diff
@@ -1650,6 +1652,37 @@ private static void sendConnectorOpToLead(LeadNodeSmartConnectorOpContext ctx)
     }
   }
 
+  public static void GET_SNAPPY_TABLE_STATS(Blob[] statsMap) throws SQLException {
+    try {
+      if (GemFireXDUtils.TraceSysProcedures) {
+        SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_SYS_PROCEDURES,
+            "executing GET_SNAPPY_TABLE_STATS ");
+      }
+      GfxdListResultCollector collector = new GfxdListResultCollector();
+      LeadNodeGetStatsMessage msg  = new LeadNodeGetStatsMessage(collector);
+      msg.executeFunction();
+      List result = (ArrayList)collector.getResult();
+      if (result != null) {
+        for (Object oneResult : result) {
+          Object o = oneResult;
+          ByteArrayOutputStream baos = new ByteArrayOutputStream();
+          try {
+            ObjectOutputStream os = new ObjectOutputStream(baos);
+            os.writeObject(o);
+            byte[] tableObjectBytes = baos.toByteArray();
```

Comment:
Done. There are a few more places Java serialization is used in procedures (thru other PRs). Will change those as well.